bundles/org.openhab.binding.sunsynk/src/main/java/org/openhab/binding/sunsynk/internal/api/AccountController.java

```diff
@@ -143,12 +149,13 @@ public ArrayList<Inverter> getDetails() throws SunSynkInverterDiscoveryException
             Gson gson = new Gson();
             Properties headers = new Properties();
             String response = "";
+
             String httpsURL = makeLoginURL(
                     "api/v1/inverters?page=1&limit=10&total=0&status=-1&sn=&plantId=&type=-2&softVer=&hmiVer=&agentCompanyId=-1&gsn=");
-            headers.setProperty("Accept", "application/json");
-            headers.setProperty("Authorization", "Bearer " + APIdata.staticAccessToken);
-            response = HttpUtil.executeUrl(HttpMethod.GET.asString(), httpsURL, headers, null, "application/json",
-                    TIMEOUT_IN_MS);
+            headers.setProperty(HttpHeaders.ACCEPT, MediaType.APPLICATION_JSON);
+            headers.setProperty(HttpHeaders.AUTHORIZATION, BEARER_TYPE + APIdata.staticAccessToken);
```

Comment:
The space between Bearer and the token is now missing.

Reply:
Good Spot